commands/init.go

```diff
@@ -27,70 +41,174 @@ var initCmd = &cmds.Command{
 		cmdkit.BoolOption(DevnetUser, "when set, populates config bootstrap addrs with the dns multiaddrs of the user devnet and other user devnet specific bootstrap parameters"),
 	},
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		repoDir := getRepoDir(req)
+		newConfig, err := getConfigFromOptions(req.Options)
+		if err != nil {
+			return err
+		}
+
+		repoDir, _ := req.Options[OptionRepoDir].(string)
```

Comment:
this is a place where we could call getrepodir with the option string and pass it into createrepo